enforcer-rules/src/it/bom-project-error/verify.groovy

```diff
@@ -28,7 +28,7 @@ Class org.apache.log.Logger is not found;
       unselected: io.grpc:grpc-core:1.17.1 (compile) / com.google.guava:guava:26.0-android (compile)
 ''')
 
-assert buildLog.contains('''[ERROR] Problematic artifacts in the dependency tree:
+assert buildLog.contains('''Problematic artifacts in the dependency tree:
```

Comment:
Because linkage problems and the dependency paths are printed as one message, this no longer carry `[ERROR] `.